docs/evaluate/temporal-cloud/gcp-private-service-connect.mdx

```diff
@@ -44,10 +56,10 @@ Set up Private Service Connect with Temporal Cloud with these steps:
 
   - For **Target service**, fill in the **Service name** with the Private Service Connect Service Name for the region you’re trying to connect to:
 
-      | Region                 | Private Service Connect Service Name                                            |
-      | ---------------------- | ------------------------------------------------------------------------------- |
-      | `us-central1`          | `projects/PROJECT/regions/us-central1/serviceAttachments/temporal-api`          |
-      | `australia-southeast1` | `projects/PROJECT/regions/australia-southeast1/serviceAttachments/temporal-api` |
+      | Region                 | Private Service Connect Service Name                                                      |
+      | ---------------------- | ----------------------------------------------------------------------------------------- |
+      | `asia-south1`          | `projects/prod-d5spc2sfeshws33bg33vwdef7/regions/asia-south1/serviceAttachments/pl-7w7tw` |
```

Comment:
@xaf Is this project string accurate? It looks like some UUIDs. Or is this an example?

Reply:
These are the correct ones!